api/store/mongo/migrations/migration_38_test.go

```diff
@@ -0,0 +1,112 @@
+package migrations
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/shellhub-io/shellhub/api/pkg/dbtest"
+	"github.com/shellhub-io/shellhub/pkg/models"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	migrate "github.com/xakep666/mongo-migrate"
+)
+
+func TestMigration38(t *testing.T) {
+	logrus.Info("Testing Migration 38")
+
+	db := dbtest.DBServer{}
+	defer db.Stop()
+
+	type Expected struct {
+		CreatedAt string
+		LastLogin string
+	}
+
+	migrations := GenerateMigrations()[37:38]
+
+	timeNow := time.Now().UTC()
+	convertDate := func(t time.Time) string {
+		return t.Format("2006-01-02 15:04:05.000")
+	}
+
+	userNoCreated := models.User{
+		ID:         "userNoCreatedID",
```

Comment:
`userNoCreatedAt` in all bellow cases; also the other example would be `userWithCreatedAt`.